addon/mixins/regenerated/models/new-platform-flexberry-g-i-s-layer-metadata.js

```diff
@@ -6,41 +6,126 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import { Projection } from 'ember-flexberry-data';
 
-/**
-  Mixin containing map layer metadata model attributes, relations & projections.
-
-  @class NewPlatformFlexberyGISLayerMetadataModelMixin
-  @extends <a href="http://emberjs.com/api/classes/Ember.Mixin.html">Ember.Mixin</a>
-*/
 export let Model = Ember.Mixin.create({
   name: DS.attr('string'),
+  description: DS.attr('string'),
+  keyWords: DS.attr('string'),
+
+  /**
+    Non-stored property.
+
+    @property anyText
+  */
+  anyText: DS.attr('string'),
```

Comment:
what is this?

Reply:
In .NET Objects assembly it is non-stored computed property for full text search.
This is a generation result, all computations are defined in related model.

Reply:
i mean that property name is not self documented, very regular